File: Eldar Dautović/ridewithme/UI/ridewithme_mobile/lib/models/chat_insert_request.dart
// -- chat_insert_request.dart --
import 'package:json_annotation/json_annotation.dart';

part 'chat_insert_request.g.dart';

@JsonSerializable()
class ChatInsertRequest {
  int? senderId;
  int? recieverId;
  String? message;

  ChatInsertRequest(
    this.senderId,
    this.recieverId,
    this.message,
  );

  factory ChatInsertRequest.fromJson(Map<String, dynamic> json) =>
      _$ChatInsertRequestFromJson(json);

  Map<String, dynamic> toJson() => _$ChatInsertRequestToJson(this);
}

File: Eldar Dautović/ridewithme/ridewithme.Model/Requests/ChatInsertRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ridewithme.Model.Requests
{
    public class ChatInsertRequest
    {
        public int SenderId { get; set; }

        public int RecieverId { get; set; }

        public string Message { get; set; }
    }
}
